README.md

```diff
@@ -1,2 +1,109 @@
+
+#### Error handling
+
+The connector will report any error at a startup in case any of auth credentials is incorrect and caused server to reject request.
+Moreover, when any of provided Push Topic does not exist, Salesforce will reject its subscription, and it will also cause connector to fail.
+Also, when `keyField` is configured but could not be found in data, error will be returned.
+There may be need to reconnect while connector is working. Server returns conditions how to reconnect, but may also terminate the connection. Such cases are also handled and connector tries to reconnect or reports the termination to the user.
+
+### Configuration Options
+
+| name              | description                                                                                                                                                                                                                                    | required | default |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `environment`     | Authorization service based on Organization’s Domain Name (e.g.: https://MyDomainName.my.salesforce.com -> `MyDomainName`) or `sandbox` for test environment.                                                                                  | `true`   |         |
+| `clientId`        | OAuth Client ID (Consumer Key).                                                                                                                                                                                                                | `true`   |         |
+| `clientSecret`    | OAuth Client Secret (Consumer Secret).                                                                                                                                                                                                         | `true`   |         |
+| `username`        | Username.                                                                                                                                                                                                                                      | `true`   |         |
+| `password`        | Password.                                                                                                                                                                                                                                      | `true`   |         |
+| `securityToken`   | Security token as described here: https://help.salesforce.com/s/articleView?id=sf.user_security_token.htm&type=5.                                                                                                                              | `false`  |         |
```

Comment:
I've been trying to come up with a suggestion to simplify this and make this connector store less sensitive info. What we essentially need is the client ID + client secret + access token. The username, password and security token are needed only in the first step, to get the refresh token. The username+password flow is something Salesforce doesn't really recommend, except in high trust environments.

We could have the connector user follow the normal OAuth flow through a browser. If we have a script which generates the authorization link, and parses the code, we wouldn't need to store the username, password and security token. WDYT? We can discuss this in a new issue, I don't think it's a blocker for this PR.

Reply:
Security-wise speaking, it is definitely a better idea to store as little sensitive data as possible 🙂 It's like You said, we only need Client ID, Client Secret and an Access Token for connector to work. I'll create an issue for that, since there are other things to discuss here.

Reply:
:+1: